src/components/Tab.vue

```diff
@@ -2,6 +2,7 @@
   <section
     v-show="isActive"
     ref="tab"
+    :id="computedId"
```

Comment:
The ID was removed due to https://github.com/Jacobs63/vue3-tabs-component/issues/22

Reply:
I understand.  However, aria-controls cannot function this way.  Therefore I have introduced a variable paneId, which adds '-pane' behind the id so that it doesn't interfere with that behavior and it doesn't jump onto the element on page load.  Is that acceptable?

Reply:
This seems like a good approach towards fixing both #31 & #34